designs/Verify-Verbosity.md

```diff
@@ -0,0 +1,454 @@
+trace:             Valid from: 4/1/2008 5:00:00 PM to 12/1/2037 3:59:59 PM
+
+Successfully verified package 'NuGet.Common.5.9.0-preview.2'.
+```
+
+</details>
+
+#### `--verbosity diagnostic`
+
+* Same output as `--verbosity detailed`
+
+### Example output in failure scenarios
+
+* Errors and Warnings are displayed to the console irrespective of `verbosity` level.
+
+#### `Verifying a tampered package`
+
+<details>
+<summary>output</summary>
+
+```
+dotnet nuget verify tampered.12.0.1.nupkg -v -n
+
+Verifying Newtonsoft.Json.12.0.3
+C:\Users\kapenaga\Downloads\tampered.12.0.1.nupkg
+Signature Hash Algorithm: SHA256
+
+Signature type: Author
+Verifying the author primary signature with certificate:
+  Subject Name: CN=Json.NET (.NET Foundation), O=Json.NET (.NET Foundation), L=Redmond, S=wa, C=US, SERIALNUMBER=603 389 068
+  SHA1 hash: 4CFB89FAA49539A58968D81960B3C1258E8F6A34
+  SHA256 hash: A3AF7AF11EBB8EF729D2D91548509717E7E0FF55A129ABC3AEAA8A6940267641
+  Issued by: CN=.NET Foundation Projects Code Signing CA, O=.NET Foundation, C=US
+  Valid from: 10/24/2018 5:00:00 PM to 10/29/2021 5:00:00 AM
+Timestamp: 11/8/2019 4:56:46 PM
+Verifying author primary signature's timestamp with timestamping service certificate:
+  Subject Name: CN=TIMESTAMP-SHA256-2019-10-15, O="DigiCert, Inc.", C=US
+  SHA1 hash: 0325BD505EDA96302DC22F4FA01E4C28BE2834C5
+  SHA256 hash: 481F4373272D98586C5364B6C115E82425675AEBFD9FACF7ADC464FA2FFFB8F0
+  Issued by: CN=DigiCert SHA2 Assured ID Timestamping CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+  Valid from: 9/30/2019 5:00:00 PM to 10/16/2030 5:00:00 PM
+
+Signature type: Repository
+Service index: https://api.nuget.org/v3/index.json
+Owners: jamesnk, newtonsoft
+Verifying the repository countersignature with certificate:
+  Subject Name: CN=NuGet.org Repository by Microsoft, O=NuGet.org Repository by Microsoft, L=Redmond, S=Washington, C=US
+  SHA1 hash: 8FB6D7FCF7AD49EB774446EFE778B33365BB7BFB
+  SHA256 hash: 0E5F38F57DC1BCC806D8494F4F90FBCEDD988B46760709CBEEC6F4219AA6157D
+  Issued by: CN=DigiCert SHA2 Assured ID Code Signing CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+  Valid from: 4/9/2018 5:00:00 PM to 4/14/2021 5:00:00 AM
+Timestamp: 11/8/2019 5:28:02 PM
+Verifying repository countersignature's timestamp with timestamping service certificate:
+  Subject Name: CN=Symantec SHA256 TimeStamping Signer - G3, OU=Symantec Trust Network, O=Symantec Corporation, C=US
+  SHA1 hash: A9A4121063D71D48E8529A4681DE803E3E7954B0
+  SHA256 hash: C474CE76007D02394E0DA5E4DE7C14C680F9E282013CFEF653EF5DB71FDF61F8
+  Issued by: CN=Symantec SHA256 TimeStamping CA, OU=Symantec Trust Network, O=Symantec Corporation, C=US
+  Valid from: 12/22/2017 4:00:00 PM to 3/22/2029 4:59:59 PM
+
+Finished with 1 errors and 0 warnings.
+error: NU3008: The package integrity check failed. The package has been tampered with since being signed.
```

Comment:
This message is slightly different from what we have on our docs: https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3008

While tampering is the correct security term, "The package has changed since it was signed" may be clearer for non-security folks. What do you think of this?

```suggestion
error: NU3008: The package integrity check failed. The package has changed since it was signed. Please ensure that the package has not been tampered.
```

Reply:
@dtivel suggested `error: NU3008: The package integrity check failed. Please ensure that the package has not been tampered since signing.` message in https://github.com/NuGet/Home/pull/10466#discussion_r591740444. Happy to engage in further conversations if we need to tweak the error message.